### DISCUSSION TITLE
Deprecate the "Select Children of Selected Instances" feature

### DISCUSSION THREAD

**Discussion by VoiditeDev** [ExperiencersInternational/auroraSuite#10] [Feature Requests]
Roblox already has a button for selecting children when you right click the selected instances. so I feel like this is redundant

Here's the video
https://github.com/ExperiencersInternational/auroraSuite/assets/142426954/5156022e-0e40-4054-9c65-750e1e22f461
[the menu doesn't appear because I use the Xbox recorder]

> **NOTE**: THIS IS IMPORTED FROM THE DEVELOPER FORUMS

**Comment by ExperiencersInternational** [comment]
Rather than depreciate it currently, I think I will instead stop displaying it as a feature on DevForum when I eventually get round to it.

That's the best option imo. I would like to figure out a way of allowing people to add external tools at launch or something, since that would be kinda cool but it'll probably never happen.
